TeaDiary/Scripts/App/Teas/TeasDetailsPage.js

```diff
@@ -0,0 +1,62 @@
+﻿$(document).ready(() => {
+    var displayer = new TeaDisplayer({
+        Name: '#Name',
+        SecondName: '#AdditionalName',
+        Type: '#Type',
+        Notes: '#Notes'
+    });
+
+    displayer.getTeaById(currentId,
+        (success) => {
+            if (!success) {
+                $(location).attr('href', TeasPageLink);
+            }
+        });
+});
+
+function showSuccessMessage() {
+    var successText = "Success";
```

Comment:
That's a good start, but it's not a constant and it's still mixed in the method and not separated